skimage/util/process.py

```diff
@@ -0,0 +1,68 @@
+from multiprocessing import cpu_count
+
+import dask.array as da
+
+__all__ = ['process_chunks']
+
+
+def _get_chunks(shape, ncpu):
```

Comment:
@cowlicks doesn't this overchunk? (that's a fun word.) e.g. if I have 4 processors, it's going to give me back 16 chunks, right? I think

1. the division should be by the ceiling of the ndim-th root of the number of cpus, and
2. In cases of non-even division, the initial chunk size should be 1 + the division result, so that you get a decent-sized remainder.

Here's what a doctest should look like imho:

```python
>>> _get_chunks((4, 4), 4)
(2, 2)
>>> _get_chunks((4, 4), 2)
(2, 2)
>>> _get_chunks((5, 5), 2)
(3, 3)
```

If we want to get more clever, this'd be a fun one:

```python
>>> _get_chunks((2, 4), 2)
(2, 2)
```